src/node_file.cc

```diff
@@ -992,6 +993,28 @@ void Close(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void FastClose(Local<Object> recv,
+                      const int32_t fd,
+                      // NOLINTNEXTLINE(runtime/references) This is V8 api.
+                      v8::FastApiCallbackOptions& options) {
+  Environment* env = Environment::GetCurrent(recv->GetCreationContextChecked());
+
+  uv_fs_t req;
+  FS_SYNC_TRACE_BEGIN(close);
+  int err = uv_fs_close(nullptr, &req, fd, nullptr) < 0;
+  FS_SYNC_TRACE_END(close);
+  uv_fs_req_cleanup(&req);
+
+  if (err < 0) {
+    options.fallback = true;
+  } else {
+    // Only remove unmanaged fds if the close was successful.
+    env->RemoveUnmanagedFd(fd);
```

Comment:
I read the source code of `RemoveUnmanagedFd`, probably this change is not a breaking change (calling it after instead of before) since the `unmanagedFd` is probably being cleaned on shutdown.